cmd/loom/dposV3_commands.go

```diff
@@ -36,6 +36,32 @@ func UnregisterCandidateCmdV3() *cobra.Command {
 	return cmd
 }
 
+func UnjailValidatorCmdV3() *cobra.Command {
+	var flags cli.ContractCallFlags
+	cmd := &cobra.Command{
+		Use:   "unjail-validator",
+		Short: "Unjail a validator",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			var validator *types.Address
+			if len(args) > 1 {
```

Comment:
Why is this `len(args) > 1`? That implies there must be two positional args, but you only look at `args[0]`

Reply:
Use `cobra.RangeArgs(0, 1)` to cap the number of args, and change the condition to `len(args) == 1`